source/util/parse_number.cpp

```diff
@@ -0,0 +1,173 @@
+// Copyright (c) 2015-2016 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Actually this is changed a little bit. We don't insert the parsed words to the instruction anymore, so we don't call `binaryEncodeU32()` and `binaryEncodeU64()` anymore. Instead we sort of inlined the 64->32 split and call emit() directly.